hack/generated/controllers/suite_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	TestNamespace          = "k8s-infra-test-ns"
-	DefaultResourceTimeout = 2 * time.Minute
+	DefaultResourceTimeout = 10 * time.Minute
```

Comment:
Is there a way to make this 10 min just for CosmosDB and keep other resources 2 min?

Reply:
@babbageclunk we already had some timeouts on master with the live tests so perhaps better to make it 10 minutes for everything? That's about the maximum we can do, BTW, go kills the tests after 11 minutes 😆